### ISSUE TITLE
Сохранение первоначального варианта при редактировании материала с модерацией

### ISSUE THREAD

**Issue by krylos** [instantsoft/icms2#265]
Т.е. в случае когда предусмотрена модерация после редактирования, модератор мог "не разрешить" изменения, тем самым восстановить первоначальное состояние материала.

**Comment by asnenne** [comment]
Согласен. Сейчас же получается, что была нормальная статья, пользователь изменил на ненормальную, модератор не принимает - остается только удалить. Но это могла быть комментируемая статья, проиндесированная, она могла давать трафик.

**Comment by Margauza** [comment]
> Т.е. в случае когда предусмотрена модерация после редактирования, модератор мог "не разрешить" изменения, тем самым восстановить первоначальное состояние материала.

"Не разрешить" или "отправить на доработку" с соответствующими пометками от модератора о причинах, а на сайте публиковать до поры прежнюю версию.
